pkg/app/src/components/Account/modules/identity-section/formik.ts

```diff
@@ -0,0 +1,40 @@
+import { Identity } from '@gamedao-haiku/graphql/dist'
+import { FormikHelpers } from 'formik'
+import * as Yup from 'yup'
+
+const webRegularExpression = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi
+export interface FormValues {
+	displayName: string
+	twitter: string
+	legalName: string
+	riot: string
+	email: string
+	totalDeposit?: string
+	web: string
+}
+
+const formikInitialValues = (identity: Identity) => ({
```

Comment:
if these are initial values, shouldnt they be empty strings by default and hydrated later?